e2e/config/azure.go

```diff
@@ -71,24 +74,24 @@ func NewAzureClient() (*AzureClient, error) {
 		// use a bunch of connections for load balancing
 		// ensure all timeouts are defined and reasonable
 		// ensure TLS1.2+ and HTTP2
-		Transport: armbalancer.New(armbalancer.Options{
-			PoolSize: 100,
-			Transport: &http.Transport{
-				Proxy: http.ProxyFromEnvironment,
-				DialContext: (&net.Dialer{
-					Timeout:   30 * time.Second,
-					KeepAlive: 30 * time.Second,
-				}).DialContext,
-				ForceAttemptHTTP2:     true,
-				MaxIdleConns:          100,
-				IdleConnTimeout:       90 * time.Second,
-				TLSHandshakeTimeout:   10 * time.Second,
-				ExpectContinueTimeout: 1 * time.Second,
-				TLSClientConfig: &tls.Config{
-					MinVersion: tls.VersionTLS12,
-				},
+		//Transport: armbalancer.New(armbalancer.Options{
```

Comment:
any reasons we are keeping those comments ?

Reply:
I'm still not fully understand if it's actually important for us or not. I think it's not, but here is a chance we will need to resurrect it in the future.

Just want to make it visible until it confirmed we don't have any issues by removing it.